getx_demo/lib/screens/loan_application_screen.dart

```diff
@@ -0,0 +1,360 @@
+                              if (account.balance.value <= 1000) {
+                                account.loanDecision.value =
+                                    LoanDecisionRules.waiting;
+                                return;
+                              }
+                              double salary =
+                                  double.parse(salaryController.value.text);
+                              if (account.hasLoan.value ||
+                                  salary <= 1000 ||
+                                  double.parse(expensesController.value.text) >=
+                                      salary / 3 ||
+                                  !accepted.value) {
+                                account.loanDecision.value =
+                                    LoanDecisionRules.declined;
+                                return;
+                              }
+                              isLoading.value = true;
+                              final response = await http.get(
+                                Uri.parse(
+                                  'http://www.randomnumberapi.com/api/v1.0/randomredditnumber?min=0&max=100&count=1',
+                                ),
+                              );
+                              isLoading.value = false;
+                              if (json.decode(response.body)[0] > 50) {
+                                account.loanDecision.value =
+                                    LoanDecisionRules.approved;
+                                account.transactions.add(
+                                  Transaction(
+                                    title: 'Loan',
+                                    amount: account.loanAmount.value,
+                                    type: TransactionType.loan,
+                                    dateTime: DateTime.now(),
+                                    id: account.uidGen.v1(),
+                                  ),
+                                );
+                                Get.defaultDialog(
+                                  title:
+                                      'Yeeeyyy !! Congrats. Your application has been approved. Don\'t tell your friends you have money!',
+                                  content: ElevatedButton(
+                                    onPressed: () => Get.offAndToNamed(
+                                        TransactionsScreen.route),
+                                    child: const Text(
+                                      'Go back',
+                                    ),
+                                  ),
+                                );
+                              } else {
+                                Get.defaultDialog(
+                                  title:
+                                      'Ooopsss. Your application has been declined. It\'s not your fault, it\'s a financial crisis.',
```

Comment:
Do localization throughout the whole application. Replace all strings with identifiers like `"application_declined".tr`, that will put the right language in runtime. Use [GetX localization](https://pub.dev/packages/get#translations).